Scripts/Spice.py

```diff
@@ -17,14 +19,185 @@ def compute_view_frustrum(instrument):
+
+def unload_all_kernels():
+    """
+    Unloads all kernels
+    """
+    spice.kclear()
+
+def check_number_kernels(kType : Kernel, value: int):
+    """
+    Check how many kernels are loaded for testing purposes
+    :param kType: Kernel enum type
+    :param value: amount of kernels that should be loaded
+    :return: boolean if there is a correct amount of kernels loaded
+    """
+    return spice.ktotal(kType) == value
+
+########
+# Time #
+########
+
+def convert_UTC_to_ET(date: datetime):
+    """
+    Converts standard UTC date into ET (Ephemeris Time)
+
+    :param date: input time in UTC form
+    :type date: datetime
+    :return: time in Ephemeris Time Format(float)
+    """
+
+    w = datetime.date.strftime(date,'%b %d, %Y')
+    return spice.utc2et(w)
+
+###############
+# SPK Kernels #
+###############
+
+def find_body_position(mainBody: str, etTime: float, referenceFrame: str, correction: str, observingBody: str):
+    """
+    Loaded from SPK file
+    Returns the position of a body relative to another observing body
+
+    Example: body_position('Cassini', 595166468.1826606, 'J2000', 'NONE', 'SATURN BARYCENTER')
+
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/spkpos_c.html
+    :param mainBody: the name of target body
+    :param etTime: time in ET format (look at convert_UTC_to_ET)
+    :param referenceFrame: string referring to the reference frame of the output position vector
+    :param correction:  Abberation correction flag
+    :param observingBody: Name of observing body
+    :return: Position of target in a 3D vector
+    :return: light time between the objects as a double
+    """
+
+    return spice.spkpos(mainBody, etTime, referenceFrame, correction, observingBody)
+
+def find_frame_transformation(fromObject: str, toObject:str, etTime: float):
+    """
+    Returns the transformation matrix of how the frame is moving
+
+    Example: find_frame_transformation ( 'IAU_EARTH', 'J2000', ET  )
+
+    ftp://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/pxform.html
+
+    :param fromObject: name of FK of which we are coming from
+    :param toObject: name of FK of which we are going to
+    :param etTime: Epoch of the rotation matrix
+    :return: ET is the amount the position has transformed (magnitude?)
+    :return: The 3x3 transformation matrix M, where (x,y,z)^T *M = (x2,y2,z2)^T
+    """
+
+    return spice.pxform(fromObject, toObject, etTime)
+
+##############
+# DSK Kernel #
+##############
+
+def find_ray_surface_intercept(method: str, target: str, etTime: float, fixedReference:str, correction: str,
+                  observerName: str, directionReference:str, directionVector: list):
+    """
+    Finds where a ray intercepts a surface
+
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/sincpt.html
+
+    :param method: computational method name such as 'ELLIPSOID'
+    :param target: name of the target body
+    :param etTime: et time of the frame
```

Comment:
Let's make this a little more explicit, and rename `etTime` to something that isn't redundant, like `et` or `time`.
```suggestion
    :param etTime: Ephemeris Time of the frame
```